### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by revmischa**
Add some documentation.
Please feel free to help make it better!

View it here: https://smorest-crud.readthedocs.io/en/doc/
Hope we can do this for more projects

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Welcome to Smorest CRUD's documentation!
+============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   source/README.md
+
```
Comment (revmischa) [inline docs/index.rst:9]: this file could be better
it renders this: https://smorest-crud.readthedocs.io/en/doc/

**Review by rkyslyy — APPROVED** [review]

File: smorest_crud/view.py [hunk 2]
```diff
@@ -14,19 +14,32 @@
...
+
+    Applies `jwt_required <https://flask-jwt-extended.readthedocs.io/en/stable/api/#flask_jwt_extended.jwt_required>`_ by default to required authenticated requests.
+    """
 
     def query(self) -> BaseQuery:
+        """Return query for `model`."""
         return self._get_model().query
 
     def query_for_user(self) -> BaseQuery:
+        """Produce a query for current model, filtered by `model.query_for_user(current_user)`."""
```
Comment (gnudeb) [inline smorest_crud/view.py:42]: Backticks do not seem to enable monospace here: 
image

**Review by gnudeb — APPROVED** [review]

File: smorest_crud/view.py (hunk 2 above)
Comment (revmischa) [reply smorest_crud/view.py:42]: I couldn't figure out what it should be

Comment (revmischa) [reply smorest_crud/view.py:42]: Maybe the theme sucks idk

**Review by Dizzzmas — APPROVED** [review]

**Review by AnneMedvedeva — APPROVED** [review]

**Review by paveu-piotrowski — APPROVED** [review]

**Review by adamszeptycki — APPROVED** [review]

**Review by janjur — APPROVED** [review]

**Review by valeriikundas — APPROVED** [review]

**Pull request merged**